pennylane/tape/tape.py

```diff
@@ -1260,6 +1285,13 @@ def num_params(self):
         """Returns the number of trainable parameters on the quantum tape."""
         return len(self.trainable_params)
 
+    @property
+    def batch_size(self):
+        r"""The batch size of the quantum tape inferred from the batch sizes
+        of the used operations for parameter broadcasting.
+        Also see :attr:`~.Operator.batch_size` for details."""
```

Comment:
```suggestion
        r"""The batch size of the quantum tape inferred from the batch sizes
        of the used operations for parameter broadcasting.

        .. seealso:: :attr:`~.Operator.batch_size` for details."""
```

Reply:
Since this is a user-facing, I would say this needs to have the details, especially the `Returns:` :)